src/CORBA/CORBACommons/CorbaAdapters.cpp

```diff
@@ -450,10 +450,12 @@ namespace CORBACommons
     return properties.size();
   }
 
+  template<typename Elem, typename Traits, typename Allocator,
```

Comment:
It is bad that we need to repeat template parameters for OutputMemoryBuffer here (and places like this).
Solution:
implement base class BaseStream with abstract 'append' method
define operators << for BaseStream class
Inherit OutputMemoryBuffer from BaseStream (and implement append in OutputMemoryBuffer)
Pass BaseStream instead OutputMemoryStream in places like this.

Reply:
done.
1) append is implemented in terms of write
2) write method of Ostream is used e.g. [Application.cpp](https://github.com/yoori/unixcommons_cobrazz/blob/c4f60326eee61afe4fdf604e3c5769a49990bb23/tests/Generics/MemBuf/Application.cpp#L88)